WAIVES.md

```diff
@@ -0,0 +1,136 @@
+# Waiving known failures
+
+## Custom file format
+
+The [waives file](conf/waives) uses a custom file format to specify when
+a failure is expected.  
+This format consists of so-called "sections", optionally separated by any
+number of empty lines. Any lines beginning with `#` are skipped.
+
+```
+<section>
+
+# some comment
+<section>
+
+...
+```
+
+Each section consists of two parts:
+
+- a list of result names, as regular expressions
+- a whitespace-indented block with a python code expression
```

Comment:
Are we talking about arbitrary indented block, or it must be 4spaces per level?

Reply:
Any arbitrary indented code block, as passed through `textwrap.dedent()`, eg. the largest whitespace indent shared by all lines in that block will be removed.
```
      a b
        c d
```
to
```
a b
  c d
```

Reply:
No need to clarify it in doc.